donate.html

```diff
@@ -2049,91 +2043,93 @@ <h4>Shipping Address</h4>
 {% braintree_js_libs %}
 <script src="/resources/ak_braintree_vzero.js"></script>
 <script>
-  $(function () {
-    {% if not page.custom_fields.donation_only_paypal or page.custom_fields.donation_only_paypal.strip|lower != 'y' %}
-    var form = document.querySelector("#act"),
-      options = {
-        form: form,
-        fields: {
-          number: {
-            selector: '#ak-card_num-hosted'
+  $(document).ready(function() {
+    $(function () {
+      {% if not page.custom_fields.donation_only_paypal or page.custom_fields.donation_only_paypal.strip|lower != 'y' %}
+      var form = document.querySelector("#act"),
+        options = {
+          form: form,
+          fields: {
+            number: {
+              selector: '#ak-card_num-hosted'
+            },
+            cvv: {
+              selector: '#ak-card_code-hosted'
+            },
+            expirationDate: {
+              selector: '#ak-exp_date-hosted',
+              placeholder: 'MM / YYYY'
+            }
           },
-          cvv: {
-            selector: '#ak-card_code-hosted'
+          styles: {
+            'input': {
+              'font-family': '{{ templateset.custom_fields.font_family|default:"sans-serif"|safe }}',
+              'font-size': '1.1rem',
+              'font-weight': 'bold',
+              'color': '#17292e',
+              'padding': '0.6em'
+            },
+            'input.invalid': {
+              'color': '{{templateset.custom_fields.color_error }}',
+              'background-color': '#fff0d4'
+            },
+            'input.valid': {
+            }
           },
-          expirationDate: {
-            selector: '#ak-exp_date-hosted',
-            placeholder: 'MM / YYYY'
-          }
-        },
-        styles: {
-          'input': {
-            'font-family': '{{ templateset.custom_fields.font_family|default:"sans-serif"|safe }}',
-            'font-size': '1.1rem',
-            'font-weight': 'bold',
-            'color': '#17292e',
-            'padding': '0.6em'
-          },
-          'input.invalid': {
-            'color': '{{templateset.custom_fields.color_error }}',
-            'background-color': '#fff0d4'
+          submitOnEmpty: function () {
+            return $('#ak-pay-by-paypal').val() == 1;

```

Comment:
Allow me to ask, why 1? Sorry If i am lacking context here.

Reply:
@mrkimani-ndegwa same response here - I haven't actually updated this code! Just copied the line from here: https://github.com/350org/ak_intl_v3/pull/305/files#diff-cd10e45c24834797bffea9a06830aa90bbbc125fbb12d9a088d04cf6a6f645d5L2085